### PR TITLE
LPD-44175 Fixed incorrect redirect after app purchase

### PR DESCRIPTION
**Pull request by CalebJamesH**
https://liferay.atlassian.net/browse/LPD-44175

### CODE REVIEW

**Comment by CalebJamesH** [comment]
ci:test:sf

**Comment by CalebJamesH** [comment]
ci:test:sf

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/GetApp/GetAppOutlet.tsx [hunk 1]
```diff
@@ -217,7 +218,9 @@ const GetAppOutlet = () => {
 				nextStepsCallbackURL
 			);
 
-			window.location.href = paymentMethodURL || nextStepsCallbackURL;
+			CommerceSelectAccountImpl.selectAccount(account?.id as number),
+				(window.location.href =
+					paymentMethodURL || nextStepsCallbackURL);
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/GetApp/GetAppOutlet.tsx:223]:
```suggestion
			await CommerceSelectAccountImpl.selectAccount(account?.id as number);
			
			window.location.href = paymentMethodURL || nextStepsCallbackURL;
```

**Review by kevenleone — CHANGES_REQUESTED** [review]

**Review by kevenleone — APPROVED** [review]

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**